pyproject.toml

```diff
@@ -107,7 +107,10 @@ allow-any-import-level = "neural_lam"
 min-similarity-lines = 10
 
 
-[tool.pdm]
+[tool.pdm.version]
+source = "scm"
+fallback_version = "0.0.0"
```

Comment:
nice! But what happens if people aren't using `pdm` but `pip` instead? Or should we just these people 👿 ?

Reply:
`pdm` is set as a backend, so installing with pip as a frontend works :)